gnomad/resources/grch38/gnomad.py

```diff
@@ -406,3 +418,72 @@ def release_vcf_path(data_type: str, version: str, contig: str) -> str:
     contig = f".{contig}" if contig else ""
     version_prefix = "r" if version.startswith("3.0") else "v"
     return f"gs://gcp-public-data--gnomad/release/{version}/vcf/{data_type}/gnomad.{data_type}.{version_prefix}{version}.sites{contig}.vcf.bgz"
+
+
+def gnomad_gks(
+    version: str,
+    variant: str,
+    data_type: str = "genomes",
+    by_ancestry_group: bool = False,
+    by_sex: bool = False,
+    vrs_only: bool = False,
+    custom_ht_path: str = None,
+) -> dict:
+    """
+    Call get_gks() and return VRS information and frequency information for the specified gnomAD release version and variant.
+
+    :param version: String of version of gnomAD release to use.
+    :param variant: String of variant to search for (chromosome, position, ref, and alt, separated by '-'). Example for a variant in build GRCh38: "chr5-38258681-C-T".
+    :param data_type: String of either "exomes" or "genomes" for the type of reads that are desired.
+    :param by_ancestry_group: Boolean to pass to obtain frequency information for each ancestry group in the desired gnomAD version.
+    :param by_sex: Boolean to pass if want to return frequency information for each ancestry group split by chromosomal sex.
+    :param vrs_only: Boolean to pass if only want VRS information returned (will not include allele frequency information).
+    :param custom_ht_path: Path of Hail Table to parse if different from what the public_release() method would return for the version.
+    :return: Dictionary containing VRS information (and frequency information split by ancestry groups and sex if desired) for the specified variant.
+
+    """
+    # Read in gnomAD release table to filter to chosen variant.
+    if custom_ht_path:
+        ht = hl.read_table(custom_ht_path)
+    else:
+        ht = hl.read_table(public_release(data_type).versions[version].path)
+
+    high_level_version = f"v{version.split('.')[0]}"
+
+    # Read coverage statistics.
+    coverage_version = version
+
+    if high_level_version == "v3":
+        coverage_version = "3.0.1"
+
+    coverage_ht = hl.read_table(coverage(data_type).versions[coverage_version].path)
```

Comment:
I remember that we chose to get rid of passing coverage_version in as an argument a little while ago. Is this setup fine, or would you like to go back to passing it in as an argument ?

Reply:
this setup is fine, we don't need to pass in a coverage version because it's dependent on the gnomAD version